infopark_webcrm_sdk.gemspec

```diff
@@ -9,13 +9,13 @@ Gem::Specification.new do |spec|
       raise "git describe failed" unless $?.success?
     end
   end
-  spec.authors       = 'Infopark AG'
-  spec.email         = 'info@infopark.de'
-  spec.summary       = 'Infopark WebCRM SDK'
+  spec.authors       = 'JustRelate'
```

Comment:
Please specify "JustRelate Group GmbH" as company name in the `spec.authors` field.